modules/virt-linux-bridge-nad-port-isolation.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-linux-bridge-nad-port-isolation.adoc_{context}"]
+= Enabling port isolation for a Linux bridge NAD
+
+You can enable port isolation for a Linux bridge network attachment definition (NAD) so that virtual machines (VMs) or pods that run on the same virtual LAN (VLAN) can operate in isolation from one another. The Linux bridge NAD creates a virtual bridge, or _virtual switch_, between network interfaces and the physical network. You can then configure port numbers on the virtual bridge so that only specific traffic can reach each port. 
+
+Isolating ports in this way can provide enhanced security, efficient resource use, and performance improvements for VM workloads that run on the same node. 
+
+.Prerequisites
+
+* For VMs, you configured either a static or dynamic IP address for each VM. See "Configuring IP addresses for virtual machines".
+* You created Linux bridge NAD by using either the web console or the command-line interface. 
+* You ensured that `spec.config.hairpinMode` in the Linux bridge NAD was not set to `true`.
+
+ts can be done statically inside the guest, or by a DHCP server in the physical network (the bridge is connected to)
```

Comment:
```suggestion
It can be done statically inside the guest, or by a DHCP server in the physical network (the bridge is connected to)
```

Reply:
Sorry leftover sentence that should have been removed. I instead point a customer to the "Configuring IP addresses for virtual machines" document.